src/Common/src/System/Security/Cryptography/Asn1/X509ExtensionAsn.manual.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Security.Cryptography.Asn1
+{
+	internal partial struct X509ExtensionAsn
+    {
+        public X509ExtensionAsn(X509Extension extension)
+        {
```

Comment:
The `skipCopy` parameter is removed and the copying of RawData is never done. The object is never used for anything but serialization with a lifetime within a single function.